src/pages/index.html

```diff
@@ -4,16 +4,380 @@
+    <form action="">
+      <input type="email" placeholder="Enter Your Email" />
+      <input type="submit" value="Start Free Trial" />
+    </form>
+    <div class="log">
+<p> Already have an account? <a href="$">Log In now</a> </p>
+
+    </div>
+  </div>
+        
+  </div>
+
+ 
+</div>
+
+  <div class="mb-3 col-xl-5 col-lg-5 col-md-5  position-relative ;
+  ">
+
+  <div class="pic">
+    <div class="shap">
+      <img src="../assets/images/shape (2).png" alt="">
+
+      
+
+              </div>
+              <div class="squr1 position-absolute">
+                <div class="d-flex">
+                  <img src="../assets/images/Icon.svg" alt="">
+                    <span class="nomber2">+3.50%</span>
+                    <i class="fa-solid fa-arrow-up"></i>
+                  </div>
+     <div class=" d-flex flex-column">
+      <span class="nomber">86,590</span>
+      <span class="total">Total Customer</span>
+              </div>
+            </div>
+
+            <div class="squr2 position-absolute ">
+              <div class="d-flex">
+                <img src="../assets/images/aaaaa.svg" alt="">
+                <span class="nomber2">+48.50%</span>
+                <i class="fa-solid fa-arrow-up"></i>
+              </div>
+   <div class=" d-flex flex-column">
+    <span class="nomber">$36,845.00</span>
+    <span class="total">Total Revenue</span>
+  </div>
+   </div>
+
+
+        
+    <img src="../assets/images/person.png" alt="">
+  </div>
+
+</div>
+
+ </div>
+</div>
+
+
+
+</div>
+
+ <div class="page">
+  <div class="container">
+    <div class="developed">
+      <div class="updated">
+        <img src="../assets/images/icon (11).svg" alt="">
+        <p>  Developed with the highest care. Always updated. </p>
+      </div>
+      <div class="updated">
+        <img src="../assets/images/icon (12).svg" alt="">
+        <p> Get free goodies from all over the world after purchase </p>
+      
+      </div>
+      <div class="updated">
+        <img src="../assets/images/icon (13).svg" alt="">
+        <p>  Save up to 10% of every purchase you made </p>
+      
+      
+      </div>
+          </div>
+    <div class="data ">
+      <div class="lysis">
+      <div class="card1" >
+      <img src="../assets/images/Graph.png" alt="">
+          <div class="shap">
+<img src="../assets/images/shape (1).png" alt="">            
+                    </div>
+       
+        
+        </div>
+      
+        <div class="card2" >
+        
+          <h4> Communicative and fast data processing</h4>
+  <p> Importance of data processing includes better increased productivity
+     and profits, target, right decisions, more accurate and reliable.  </p>
+     <a href="#">  Make your first sale <i class="fa-solid fa-arrow-right"></i> </a>
+
+        </div>
+  
+      </div>
+      </div>
+
+   
+
+    <!-- <img src="../assets/images/Vector (1).svg" alt=""> -->
+
+
+
+
+</div>
+<div class="Payment">
+
+      
+  <div class="easiest  ">
```

Comment:
![344335854-be94c462-7ea4-47aa-916c-16155e92b1c3](https://github.com/shaban011147/landing/assets/106445705/ec799fc9-41a5-4dd9-a415-87c04998cdd0)

1-where space between img (card1) and (card2)

Reply:
done